cranelift/entity/src/set.rs

```diff
@@ -254,4 +248,27 @@ mod tests {
 
         assert!(m.is_empty());
     }
+
+    #[test]
+    fn cardinality() {
+        let mut m = EntitySet::new();
+
+        m.insert(E(0));
+        m.insert(E(1));
+        assert!(m.cardinality() == 2);
+
+        m.insert(E(1));
+        assert!(m.cardinality() == 2);
+
+        m.insert(E(42));
+        assert!(m.cardinality() == 3);
+
+        assert!(m.pop() == Some(E(42)));
```

Comment:
Would you make this case be something greater than 64 instead of 42? That way it tests when `len` grows to a larger number of `usize` elements, and also when it shrinks back to the smaller number of elements.

If you make it 100, then we can also test that the values added below don't get double-counted when the underlying vector shrinks and re-grows.